rmgpy/cantherm/statmech.py

```diff
@@ -537,18 +537,17 @@ def applyEnergyCorrections(E0, modelChemistry, atoms, bonds):
     # Spin orbit correction (SOC) in Hartrees
     # Values taken from ref 22 of http://dx.doi.org/10.1063/1.477794 and converted to hartrees
     # Values in millihartree are also available (with fewer significant figures) from table VII of http://dx.doi.org/10.1063/1.473182 
-    SOC = {'H':0.0, 'N':0.0, 'O': -0.000355, 'C': -0.000135, 'S':  -0.000893, 'P': 0.0} 
+    SOC = {'H':0.0, 'N':0.0, 'O': -0.000355, 'C': -0.000135, 'S':  -0.000893, 'P': 0.0, 'I':-0.011547226,}
```

Comment:
I didn't see the SOC value for iodine in the ref in the comments above. Please add a new ref or add the method of calculation.

Reply:
@phalgunlolur , could you see where did this SOC data come from?